maxim/models/prompt.py

```
@@ -180,6 +180,7 @@ class PromptResponse:
     model_params: Dict[str, Union[str, int, bool, Dict, None]] = field(
         default_factory=dict
     )
+    resolved_messages: Optional[List[ChatCompletionMessage]] = None
```

Ruff (0.14.10)

Undefined name `ChatCompletionMessage`

(F821)